packages/pds/src/sql-blockstore.ts

```diff
@@ -27,18 +31,21 @@ export class SqlBlockstore extends IpldStore {
   }
 
   async putBytes(cid: CID, bytes: Uint8Array): Promise<void> {
-    const row = {
-      cid: cid.toString(),
-      did: this.did,
-      size: bytes.length,
-      content: bytes,
-      indexedAt: new Date().toISOString(),
-    }
-    await this.db.db
+    const insertBlock = this.db.db
```

Comment:
Only if you're feelin it.
```suggestion
    this.db.assertTransaction()
    const insertBlock = this.db.db
```

Reply:
hmmm yeah that probably makes sense 🤔 

if for some reason we find we need to do this non-transactionally, we can loosen the constraint